sidebars.js

```diff
@@ -252,6 +252,13 @@ module.exports = {
         }
       ]
     },
+    {
+      type: 'category',
+      label: 'Advanced Tutorials',
```

Comment:
While it might seem at odds with the "4 types of docs" principles, I contend that this article best belongs in `How to Guides > Creating & Editing Expectations > Advanced`. Additionally, adding a top-level category dilutes out the value of the existing categories.